src/ports/postgres/modules/graph/wcc.py_in

```diff
@@ -289,6 +291,9 @@ def wcc(schema_madlib, vertex_table, vertex_id, edge_table, edge_args,
                                 SELECT COUNT(*) AS cnt FROM {toupdate}
                             """.format(**locals()))[0]["cnt"]
 
+    if not is_platform_pg():
+        # Drop intermediate table created for Greenplum
+        plpy.execute("DROP TABLE IF EXISTS {edge_inverse}".format(**locals()))
```

Comment:
`plpy.execute("DROP TABLE IF EXISTS {0}".format(edge_inverse))` might be better. I can make the change before merging this PR.